irr/src/CGUISkin.cpp

```diff
@@ -4,7 +4,7 @@
 // This file is part of the "Irrlicht Engine".
 // For conditions of distribution and use, see copyright notice in irrlicht.h
 
-#include "guiSkin.h"
+#include "CGUISkin.h"
```

Comment:
this would conflict with this change btw ad3f2b7fed4a7ddcd4664290ecae0086b4103230

I know it's more work but maybe we should be moving the implementation to `src/gui/` and deleting the one in irrlicht

Reply:
we could also just ignore this however. the commit I linked causes more problems anyway so I might just drop it

Reply:
I got rid of the CGUISkin.h include in guiButton.cpp